.teamcity/settings.kts

```diff
@@ -87,7 +87,7 @@ project {
             schedulingPolicy = daily {
                 hour = 2
             }
-            branchFilter = "+:refs/head/main"
+            branchFilter = "+:refs/heads/main"
```

Comment:
❔Was this the reason why the daily trigger didn't work?

Reply:
Yes, it took me a while to notice the typo 😭

Reply:
I also did not spot it.